api_tests/src/comment.spec.ts

```diff
@@ -740,3 +742,37 @@ test("Report a comment", async () => {
   );
   expect(betaReport.reason).toBe(alphaReport.reason);
 });
+
+test("Dont send a comment reply to a blocked community", async () => {
+  let newCommunity = await createCommunity(beta);
+  let newCommunityId = newCommunity.community_view.community.id;
+
+  // Create a post on beta
+  let betaPost = await createPost(beta, newCommunityId);
+
+  let alphaPost = (await resolvePost(alpha, betaPost.post_view.post))!.post;
+  if (!alphaPost) {
+    throw "unable to locate post on alpha";
+  }
+
+  // Beta blocks the new beta community
+  let blockRes = await blockCommunity(beta, newCommunityId, true);
+  expect(blockRes.blocked).toBe(true);
+  delay();
+
+  // Alpha creates a comment
+  let commentRes = await createComment(alpha, alphaPost.post.id);
+  expect(commentRes.comment_view.comment.content).toBeDefined();
+  await resolveComment(beta, commentRes.comment_view.comment);
+
+  // Check beta's inbox count, make sure it stays the same
+  let unreadCount = await getUnreadCount(beta);
+  expect(unreadCount.replies).toBe(1);
+
+  let replies = await getReplies(beta);
+  expect(replies.replies.length).toBe(1);
```

Comment:
Should also check these counts before the comment is created to make sure it stays the same. And ensure that `resolveComment` was actually successful.